2nd-gen/packages/swc/.storybook/preview.ts

```diff
@@ -55,7 +55,7 @@ const preview = {
             page: DocumentTemplate,
             toc: {
                 contentsSelector: '.sbdocs-content',
-                headingSelector: 'h2, h3',
+                headingSelector: 'h2:not(.demo), h3:not(.demo), h4:not(.demo)',
                 ignoreSelector: '.sbdocs-subtitle',
```

Comment:
Or could we just ignore anything in a story canvas?

Reply:
ooooo maybe we can add demo to ingnore selector